packages/vrm-viewer/types-shim/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// we should remove this once upstream supports
```

Comment:
generated with `build-types`

Reply:
https://github.com/pixiv/three-vrm/pull/1395#pullrequestreview-2073002062
1. model-viewer was not compliant with three v164 in `USDZExporter`, fix has been merged by not released
2. if we run tsc it can generate dts file but raise error
3. as experimental we do not care error in model-viewer at this point so commit it as shim
4. eslint is not related.

Reply:
you are talking about `types-shim/features/vrm-mixin.d.ts:14` right?

Reply:
No that's irrelevant. Just run `build-types` and you'll get
```
$ tsc --declaration --declarationDir ./types --emitDeclarationOnly
../../node_modules/@google/model-viewer/src/features/ar.ts:449:42 - error TS2554: Expected 3-4 arguments, but got 1.

449       const arraybuffer = await exporter.parse(model);
                                             ~~~~~

  ../../node_modules/@types/three/examples/jsm/exporters/USDZExporter.d.ts:13:9
    13         onDone: (result: Uint8Array) => void,
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    An argument for 'onDone' was not provided.

../../node_modules/@google/model-viewer/src/features/ar.ts:454:30 - error TS2322: Type 'void' is not assignable to type 'BlobPart'.

454       const blob = new Blob([arraybuffer], {
                                 ~~~~~~~~~~~


Found 2 errors in the same file, starting at: ../../node_modules/@google/model-viewer/src/features/ar.ts:449

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Thanks. Would you put [a link to the comment above](https://github.com/pixiv/three-vrm/pull/1395#discussion_r1621561490) as a comment?

Reply:
https://github.com/pixiv/three-vrm/pull/1395/commits/5f72879101b41d28aeb2058e912ff5ea24990ac0